### DISCUSSION TITLE
How to hide user menu dropdown-header?

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#12570] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.72

### How can we help you?

I want to hide the user menu dropdown-header is there any method or something like that?

<img width="227" alt="Screenshot 2024-05-01 at 13 30 52" src="https://github.com/filamentphp/filament/assets/121255432/d012d912-105d-45cf-8b29-21a59ad7e34b">

**Comment by xalabama** [comment, marked as answer]
If i remember correctly, there is no panel setting for this. If you want to change the behavior of displaying the user menu, you have to publish the view files of filament with `php artisan vendor:publish` and then select the `filament-panels-views`. Inside the `resources/views/vendor/filament-panels` directory, you now will find the view file that controls this behavior. Its named `user-menu.blade.php`. 

If you have disabled the profile feature in general, you can simply change this line from `@if ($profileItem?->isVisible() ?? true)` to `@if ($profileItem?->isVisible() ?? false)`. 

Otherwise you could also remove all content of the if statement starting from `@if ($profileItem?->isVisible() ?? true)`. 

Hope this will help you! :)

**Reply by devhoussam1998** [reply]
thanks @xalabama

**Comment by informatJonas** [comment]
Wouldn't that be worth an issue? - I have just searched and debugged this too.

It's also pretty weird, because if you give the UserMenu a profile item and then set it to `hidden()`, it is no longer passed to the blade file as a userMenuItem.
